.storybook/addons.js

```diff
@@ -0,0 +1,2 @@
+import '@storybook/addon-knobs/register';
+import '@storybook/addon-options/register';
```

Comment:
Newline missing and as I said in the other review: no biggie, fix if you want.

Reply:
```suggestion
import '@storybook/addon-options/register';

```